### PR TITLE
chore: add test and format

### PR DESCRIPTION
**Pull request by michael-siek**
I would add more test but I was unsure of what args was required for `neo_legacy`

### CODE REVIEW

**Review by lllwvlvwlll — CHANGES_REQUESTED** [review]

File: src/constants/index.ts [hunk 1]
```diff
@@ -1 +1,18 @@
-export const DORA_URL = "https://dora.coz.io"
\ No newline at end of file
+import type { AxiosInstance } from 'axios'
+import axios from 'axios'
+
+export const DORA_URL = 'https://dora.coz.io'
+
+export const AXIOS_DORA = (version: string): AxiosInstance => {
```
[the remark this answers is not in the document]
Comment (michael-siek) [reply src/constants/index.ts:6]: @lllwvlvwlll do you mean that the api should look like `http://dora.coz.io/api/v2/neo2` and `http://dora.coz.io/api/v3/neo3`?

[the remark this answers is not in the document]
Comment (lllwvlvwlll) [reply src/constants/index.ts:6]: The endpoint is fine.  I just mean that the method should be like this: 
```ts
export const AXIOS_DORA = (protocol: string): AxiosInstance => {
  let endpoint: string = ''

  if (protocol=== neoLegacy') {
    endpoint = '/api/v1/neo2'
  } else if (protocol=== 'neo') {
    endpoint = '/api/v1/neo3'
  }

  return axios.create({
    baseURL: DORA_URL + endpoint
  })
}
```

:)

[the remark this answers is not in the document]
Comment (michael-siek) [reply src/constants/index.ts:6]: all changed and ready for review again :grin:

[the remark this answers is not in the document]
Comment (michael-siek) [reply src/constants/index.ts:6]: sorry its fixed now apparently changes don't happen when you don't push ¯\_(ツ)_/¯

**Review by lllwvlvwlll — APPROVED** [review]

**Pull request merged**